beta/project-griffon/using-project-griffon/adobe-analytics-and-project-griffon.md

```diff
@@ -6,7 +6,21 @@ The integration with Adobe Analytics provides a richer view of SDK events to use
 
 ![](../../../.gitbook/assets/aa-loop.gif)
 
-If the SDK made a network request with Adobe Analytics \(queued\), and if post-processing information is returned from Adobe Analytics about that event \(validated\), the status tells you when an SDK event is generated \(processed\).
+### Post-processed status
+
+If the SDK made a network request with Adobe Analytics, and if post-processing information is returned from Adobe Analytics about that event, the status tells you when an SDK event is generated.
+
+| Status | Description |
+| :--- | :--- | 
+|`Queued`|Network request is being made to fetch the post-processing information.|
+|`Processed`|Network request is successful and post-processing information is received.|
+|`Delayed`|Exceeded the maximum network request retries to fetch the post-processing information.|
+|`Error`|Error casued the network request to fail. More details about the error is displayed in the event details view.|
+|`Unauthorized`|Failed to retrieve post-processed data because the user does not have access to the Analytics report suite.|
```

Comment:
"The user does not have access to the Adobe Analytics report suite." 

(I don't think we need to mention that it's failed to retrieve? But if we want to include that information, I'd go with "The user does not have access to the Adobe Analytics report suite. As a result, failed to retrieve the post-processed data.", since the lack of user access is (IMO) more important than the failure to retrieve.)